### PR TITLE
Add WSPP

### PR DESCRIPTION
**Pull request by wolfsafepoorpeople**

### CODE REVIEW

**Comment by wolfsafepoorpeople** [comment]
Payment Fee done :
https://explorer.kardiachain.io/tx/0xe0066b481ec6dcb345a20cbf5bdc0091734bc30b735ef438c71065c4b8f24c95

File: tokens/0xccB32737C6dFfddFFB24CA8A96b588ac7b1822e7/data.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+	"contract": "0xccB32737C6dFfddFFB24CA8A96b588ac7b1822e7",
+	"type": "KRC-20",
+	"name": "WolfSafePoorPeople",
+	"symbol": "WSPP",
+	"decimals": 18,
+	"description": "WSPP This is the first currency that has a program to reduce world poverty, first launched on Binance Smart Chain, but now on Polygon, Fantom, Avalanche & KardiaChain. The establishment of the Wolfible website aims to develop the Wolf safe poor people (WSPP) project in the NFT marketplace & DeFi sector.",
+	"website": "https://wolfsafepoorpeople.com",
```
Comment (phongnhat19) [inline tokens/0xccB32737C6dFfddFFB24CA8A96b588ac7b1822e7/data.json:8]: Website is not accessible. Please update.

**Review by phongnhat19 — CHANGES_REQUESTED** [review]

File: tokens/0xccB32737C6dFfddFFB24CA8A96b588ac7b1822e7/data.json (hunk 1 above)
Comment (wolfsafepoorpeople) [reply tokens/0xccB32737C6dFfddFFB24CA8A96b588ac7b1822e7/data.json]: Done please Check

Comment (phongnhat19) [reply tokens/0xccB32737C6dFfddFFB24CA8A96b588ac7b1822e7/data.json]: Still can't be access in my region. Do I need to use some kind of VPN ?

Comment (wolfsafepoorpeople) [reply tokens/0xccB32737C6dFfddFFB24CA8A96b588ac7b1822e7/data.json]: please check my commit 2 
brother i change website to https://wolfible.com

if you want to accress wolfsafepoorpeople should use VPN

Comment (wolfsafepoorpeople) [reply tokens/0xccB32737C6dFfddFFB24CA8A96b588ac7b1822e7/data.json]: > 
> 
> Still can't be access in my region. Do I need to use some kind of VPN ?

i have change the data.json 
you can check on commit 2 
the website is change to new wolfible website brother

**Review by wolfsafepoorpeople — COMMENTED** [review]
this is i change the website

**Comment by wolfsafepoorpeople** [comment]
Please Check it

**Review by phongnhat19 — APPROVED** [review]

**Pull request merged**